lua/entities/gmod_tardis/modules/sh_health.lua

```diff
@@ -0,0 +1,174 @@
+--Health
+
+CreateConVar("tardisrw_maxhealth", 1000, {FCVAR_ARCHIVE, FCVAR_REPLICATED}, "TARDIS Rewrite - Maximum health")
```

Comment:
Can these please be renamed to `tardis2_...` I assume this is named as such due to conflict with original addon. Don't really want many refs to rewrite in code so that if it becomes "complete" I can just change the name of the addon and a couple of tiny things and job done. Also remove "rewrite" from the convar description